src/coreclr/jit/jiteh.cpp

```diff
@@ -3499,12 +3498,6 @@ void Compiler::fgVerifyHandlerTab()
 #endif // FEATURE_EH_FUNCLETS
         }
 
-        // Only the first block of 'try' regions should have BBF_TRY_BEG set.
-        if (!blockTryBegSet[block->bbNum])
```

Comment:
The only use of `blockTryBegSet` currently is in this check. You could remove it entirely.

Reply:
Deleted.